### DISCUSSION TITLE
Mesh assigned node ID 0 which is different from the (zero-based) Exodus ID -1!

### DISCUSSION THREAD

**Discussion by Krystalfang** [idaholab/moose#21352] [Q&A Meshing] [answered]
I'm having trouble computing a complex model with moose, and the error message is as follows:

*** ERROR ***
Error!  Mesh assigned node ID 0 which is different from the (zero-based) Exodus ID -1!

I think this problem comes from my pre-processing process. I completed the pre-processing on cubit, but I don't know exactly what this error message is pointing to, how do I need to modify it, or redefine the node ID.

**Comment by GiudGiud** [comment, marked as answer]
Hello

This is coming from the exodus file reader in libmesh. I think it s being confused because the numbering in your exodus file is not as expected by the reader. 
Did you renumber node ids? What indexing did you choose?

Guillaume

**Reply by Krystalfang** [reply]
I'm not renumbering the nodes, there are 30,000+ output nodes,  I think it is hard.

**Reply by Krystalfang** [reply]
I see that there is a redefinition of node ID in cubit, what should be the starting ID recognized by libmesh.
![image](https://user-images.githubusercontent.com/107802065/174510776-e56584dc-5b42-44d0-90fb-507f77c9f687.png)

**Reply by GiudGiud** [reply]
Try 0

**Reply by GiudGiud** [reply]
any updates on this?
What is the numbering of the nodes when you open the file with another software than MOOSE

**Reply by Krystalfang** [reply]
Hi, thank you very much for your attention and sorry for being late. After following your suggestion, I redefine the node number in Trelis, starting from 0. From the performance of MOOSE, this problem solved successfully.
